README.md

```diff
@@ -247,6 +247,7 @@ module "enrich_kinesis" {
 | <a name="input_ssh_key_name"></a> [ssh\_key\_name](#input\_ssh\_key\_name) | The name of the SSH key-pair to attach to all EC2 nodes deployed | `string` | n/a | yes |
 | <a name="input_subnet_ids"></a> [subnet\_ids](#input\_subnet\_ids) | The list of subnets to deploy Enrich across | `list(string)` | n/a | yes |
 | <a name="input_vpc_id"></a> [vpc\_id](#input\_vpc\_id) | The VPC to deploy Enrich within | `string` | n/a | yes |
```

Comment:
There was an update in the variables.tf for the description, but it wasn't reflected in the readme.

Is possible to use a tool to generate the tables: terraform-docs
Check example usage here: https://github.com/snowplow-devops/terraform-module-template